ReadMe.md

```diff
@@ -43,6 +43,18 @@ Building Perfharness in Eclipse requires some extra dependancies importing to th
     * jms.jar
 * ant-contrib.jar (Always required)
 
+### AMQP Channel Module Pre-reqs
+
+Additional Pre-reqs for AMQP Channel Module
+
+* The Java AMQP Client required by this module can be download from 
+https://developer.ibm.com/messaging/ibm-mq-light-downloads and selecting "Download from Maven Central".
+Refer to the AMQP Client READ.md as further pre-reqs jars are required.
+
+* The allclient.jar is required from the a MQ V9.0 installation.
+
+* Locate all the pre-req jars in located in /lib of the AMQPPerfharness.

```

Comment:
I don't think that we are able to include these JAR files here... instead we typically need to point to where to acquire them such that a user can accept any associated license